From 4b8db54b926b61e465c1695d0598b4b7fd779bb8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Alex=20R=C3=B8nne=20Petersen?= <alex@alexrp.com>
Date: Mon, 10 Nov 2025 12:54:04 +0100
Subject: [PATCH 051/133] llvm: set sub-arch for spirv 1.6
X-Developer-Signature: v=1; a=openpgp-sha256; l=650; i=xtex@astrafall.org;
 h=from:subject; bh=2a4Bpp7IAYTvQbj4/oft3f9gfsDhfb9Ea7f2wfQquBk=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDJn78hj0emPrZ//48qGbX1ft655t88R5/ylzPWNovsrlL
 HJg58G1HRNZGMS4GCzFFFmKDBu8WXXS+UWXlcvCzGFlAhkiLdLAwMDAwMLAl5uYV2qkY6Rnqm2o
 Z2ioY6RjxMDFKQBTrX+c4Z99r9teW/2de2K5O5lkfYvPOe7QXNXz1KL3r7tE1ru98UmMDKuZFKf
 XfDGatuWoXFXNgz9rQ9dcSzNsLYtQmhogdu7eT0YA
X-Developer-Key: i=xtex@astrafall.org; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

(cherry picked from commit cde865e06afa772983a3833920369bed5d995f8b)
---
 src/codegen/llvm.zig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/codegen/llvm.zig b/src/codegen/llvm.zig
index b2f1518caa07..88d3a2e05cfe 100644
--- a/src/codegen/llvm.zig
+++ b/src/codegen/llvm.zig
@@ -154,6 +154,7 @@ pub fn targetTriple(allocator: Allocator, target: *const std.Target) ![]const u8
             .{ .spe, "spe" },
         }),
         .spirv32, .spirv64 => subArchName(target, .spirv, .{
+            .{ .v1_6, "1.6" },
             .{ .v1_5, "1.5" },
             .{ .v1_4, "1.4" },
             .{ .v1_3, "1.3" },
-- 
2.52.0

